src/pybind/mgr/cephadm/serve.py

```diff
@@ -311,6 +319,16 @@ def refresh(host: str) -> None:
                 if r:
                     bad_hosts.append(r)
 
+            if self.mgr.cache.host_needs_custom_logrotate_file(host):
+                for logrotate_type in [LogrotateConfigType.cephadm, LogrotateConfigType.cluster]:
+                    if self.mgr.get_store(f'{str(logrotate_type)}_logrotate_config'):
```

Comment:
Since you have to construct this string in mulitple places it might be less chances of future copy-n-paste errors to do it as a method (or property) or LogrotateConfigType. Example:
```
class LogrotateConfigType(str, Enum):
    #... values ...
    
    def config_key(self) -> str:
        return f"{self.value}_logrotate_config"
```